### ISSUE TITLE
Use Katex and pandoc markdown

### ISSUE THREAD

**Issue by ianchanning** [davidgranstrom/nvim-markdown-preview#3]
This plugin is working wonderfully, but seeing as you're using pandoc to convert the markdown it's a shame not to use [pandoc markdown](https://pandoc.org/MANUAL.html#pandocs-markdown). On top of that there is the wonderful [Katex library](https://katex.org/) for displaying LaTeX math in HTML and pandoc has native support for this with the `--katex` param.

I use the following command to get some beautiful pandoc HTML out:

```sh
pandoc -f markdown test.md -o test.html --standalone -t html --katex
```
Looking at these lines https://github.com/davidgranstrom/nvim-markdown-preview/blob/d7cbefc69f93f55e376c4cb9d754390028659ea2/autoload/markdown.vim#L24-L36

It looks like you use the GitHub formatted markdown by default. You could also set the markdown `--from` parameter to be an option.

**Comment by davidgranstrom** [comment]
Hi @ianchanning,

This sounds like a great addition to me! I would be very open to a PR for this feature if you're feeling up to it.

**Comment by davidgranstrom** [comment]
Resolved with #6
